HTML/CSS & JavaScript capstone project - Conference page

jovasal opened the pull request:
<h1> HTML/CSS & JavaScript capstone project - Conference page </h1>

- Understand how to use medium-fidelity wireframes to create a UI.
- Follow written requirements (e.g. user stories).
- Use semantic HTML tags.
- Apply best practices in HTML code.
- Use CSS selectors correctly.
- Use CSS box model.
- Use Flexbox to place elements in the page.
- Use images and backgrounds to enhance the look of a website.
- Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
- Use GitHub Pages to deploy web pages.
- Apply JavaScript best practices and language style guides in code.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Use objects to store and access data.
- Communicate technical concepts to other technical people.

jovasal committed:
test.md

test

jovasal committed:
instaling linters and coping readme and lisences files

done

jovasal committed:
firs part almos complete

jovasal committed:
background second part

image

jovasal committed:
Main Program

Main Program Complee

jovasal committed:
linters errors fixed

linters cleaned

jovasal committed:
images size

images

jovasal committed:
gap uploaded

gad fixed

jovasal committed:
change in legend

legends

jovasal committed:
checking pictures and menu

menu

jovasal committed:
linters reemplaced for JavaScript and Menu is working

all ok

jovasal committed:
speakers Dinamic

ok

jovasal committed:
details complete and responsive

responsive

jovasal committed:
Rename Index.html to index.html

jovasal committed:
sponsor done

done

jovasal committed:
Merge branch 'Mobile' of https://github.com/jovasal/Capstone into Mobile

jovasal committed:
details and readme

readme

jovasal committed:
video

video

jovasal committed:
linters errors

linters woeking fine

Cabraham1 commented on README.md:
```diff
@@ -1 +1,198 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Thank you so much to Miroverse course and FIGMA
```
nice work so far 👍🏿  i n this section it is recommended to give acknowledgment to the author of the design  [Cindy Shin](https://www.behance.net/adagio07)

jovasal replied:
Great!!

it was changed for:

Original design idea by Cindy Shin: https://www.behance.net/adagio07

Cabraham1 commented on index.html:
```diff
@@ -0,0 +1,288 @@
+    </head>
+    <body class="bodyart">
+        <header id="header" class="header">
+            <nav class="navbar">
+                <ul class="nav-list">
+                    <li>
+                        <img src="./images/Rectangle.png" class="facebook" alt="facebook"/>
+                    </li>
+                    <li>
+                        <img src="./images/twitter.png" class="facebook" alt="facebook"/>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">English</a>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">My Page</a>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">Logout</a>
+                    </li>
+                </ul>
```
Nice work so far with your project 👍🏿  but I notice your page only has one page which is the index as a project requirement you are to create two pages for this project the index page and the about us page please do well to create the about us page and all its corresponding layout.
- [x] Also in the new page about us page please add the correct navigation to be about to go back and forth of the home and About pages

jovasal replied:
About Us was created successfully as you can see in the following image:

![image](https://github.com/jovasal/Capstone/assets/35693330/9a98149b-49ee-4131-94ac-949ae4e70fc9)

Cabraham1 commented on index.html:
```diff
@@ -0,0 +1,288 @@
+                        <div class="SpeakerCurr">
+                            <h2 class="SpeakerTitle">Edmond Dantes</h2>
+                            <p class="SpeakerPosition">Account Manager</p>
+                            <p class="SpeakerDescription">Introducing Edmond, an accomplished account manager specializing in the realm of e-commerce. With a keen eye for detail and exceptional client relationship skills, Edmond ensures seamless operations and client satisfaction.</p>
+                        </div>
+                    </div>
+                    <div class="speakerInfo">
+                        <div class="SepeakerPicture">
+                            <img src="./images/speaker_background.jpg" class="speakerBackground" alt="speakerBackground">
+                        <img src="./images/pexels-andrea-piacquadio-3769021.jpg"  class="speakerImg" alt="speakerImg">
+                        </div>
+                        <div class="SpeakerCurr">
+                            <h2 class="SpeakerTitle">Andrea Piaquadio</h2>
+                            <p class="SpeakerPosition">Project Manager</p>
+                            <p class="SpeakerDescription">Say hello to Andrea, a skilled project manager in the realm of e-commerce. With a knack for organization and a strategic mindset, Andrea ensures successful implementation and delivery of projects in the digital marketplace.</p>
+                        </div>
+                    </div>
+                    <div class="speakerInfo">
+                        <div class="SepeakerPicture">
+                            <img src="./images/speaker_background.jpg" class="speakerBackground" alt="speakerBackground">
+                        <img src="./images/pexels-emmy-e-2381069.jpg"  class="speakerImg" alt="speakerImg">
+                        </div>
+                        <div class="SpeakerCurr">
+                            <h2 class="SpeakerTitle">Alex Wolf</h2>
+                            <p class="SpeakerPosition">Human Recourses</p>
+                            <p class="SpeakerDescription">Meet Alex, a dedicated human resources professional specializing in the fast-paced world of e-commerce. With a focus on talent acquisition and employee development, Alex cultivates a thriving and motivated workforce within the industry.</p>
+                        </div>
+                    </div>
+                    <div class="speakerInfo">
+                        <div class="SepeakerPicture">
+                            <img src="./images/speaker_background.jpg" class="speakerBackground" alt="speakerBackground">
+                        <img src="./images/pexels-christina-morillo-1181690.jpg"  class="speakerImg" alt="speakerImg">
+                        </div>
+                        <div class="SpeakerCurr">
+                            <h2 class="SpeakerTitle">Christina Morillo</h2>
+                            <p class="SpeakerPosition">Finances</p>
+                            <p class="SpeakerDescription">Introducing Christina, a dynamic professional specializing in business deployment within the realm of e-commerce. With a strategic mindset and meticulous planning, Christina drives successful implementation of business strategies in the digital marketplace.</p>
+                        </div>
+                    </div>
+                </div>
```
- [x] NIce so far: as per project requirement, you are not to render the future speaker on the index pages you are expected to render it f dynamically from your javascript using the array method. i kindly suggest your render by dynamically from javascript

jovasal replied:
ok. I restarted again in speakers section. Now is working with DOM

Cabraham1 reviewed (changes requested):
# Required Changes :recycle:
![almost-there](https://user-images.githubusercontent.com/65860678/170491251-b1add8ba-b18f-4648-a893-20e510d957fd.gif)

## Hey @jovasal  :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:
- [ ] I kindly request that you adhere to the Figma design provided for your project. The Figma design serves as a visual reference and guideline for implementing the desired user interface and overall look of your application. Please ensure that the implemented code aligns with the design specifications provided in Figma. This includes aspects such as layout, color scheme, typography, and any other visual elements defined in the design.

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers, and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jovasal committed:
change in speakers section usin DOM and array methoid

is working fine

jovasal committed:
linters errors

spaces and ++

danmainah commented on index.html:
```diff
@@ -0,0 +1,278 @@
+                    <div class="siteDir1">ccsummit2015@anaxtasia.com</div>
+                    <div class="ContainerSectionSpeakers">
+                        <div class="SectionAboutUs" id="SectionAboutUs">
+                            About Us
```
- [x] @jovasal as previously requested kindly separate the About page from the index page and ensure somebody can navigate from the homepage to the About page and vice versa.

jovasal replied:
Done. I made a section About, on a different page.

![image](https://github.com/jovasal/Capstone/assets/35693330/44d76392-c660-4d64-9c38-5236ec77a914)

danmainah reviewed (changes requested):
Hi @jovasal , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️  Implemented change requested to render the speaker section dynamically.

 # Required Changes ♻️

- [x] @jovasal as the previous review noted most of the sections don't match the Figma design given, below is how the pages should look, we are only allowed to change photos and background only, the look should match the design given to the check the comparison below



Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/jovasal/Capstone/assets/61408860/fae825e5-1d92-4855-a561-d7c890403f08)  | ![image](https://github.com/jovasal/Capstone/assets/61408860/41344b63-27cd-4e8c-b073-9d53fa935060)


Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/jovasal/Capstone/assets/61408860/8ee33b7d-9f5d-4ad5-b38e-b3b8d932a5d1)  | ![image](https://github.com/jovasal/Capstone/assets/61408860/fe7350d2-f2b7-4a25-9699-21885e91f4fb)



_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jovasal committed:
Change requiered under request review

changes done

jovasal committed:
fixed linters errors

Now, is posible to send again

Alfredbis29 reviewed (changes requested):
#  STATUS: REQUIRED CHANGES 
 ## Hi @jovasal 
 Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ![](https://camo.githubusercontent.com/dbe642c07a12e38cdc021eb826931557cc1eb657023c3609cc10e8f3b47f06d7/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6c344a7a686a56435169576765414f46612f67697068792e676966) 
 ## Highlights 
:heavy_check_mark:  Your Work looks Professional 
:heavy_check_mark:  Nice work so far :+1: 
 
 ## Required Changes 

- [ ] @jovasal  I notice you have done excellent job thus far, but there is still something you need to improve on, as requested by the previous reviewer[here](https://github.com/jovasal/Capstone/pull/1#pullrequestreview-1517086178) , Your design should be consistent with the Template Design; please fix this for your project to meet the standards. :wink: 

> Behance Design
>  ![image](https://user-images.githubusercontent.com/61408860/251542533-fae825e5-1d92-4855-a561-d7c890403f08.png)	

Your design
![Screenshot from 2023-07-06 23-56-04](https://github.com/jovasal/Capstone/assets/88894525/79fc1fd6-1cff-4c9a-9b3c-945e96fc225e)

> 

- [x]  Compare your Design to the Original Design for this Project are very different as mentioned by the previous reviewer , your design should refect the original design , Kindly check the vedeo below and match it to the design given for this Project :+1:   **we are only allowed to change photos and background only**

https://github.com/jovasal/Capstone/assets/88894525/1f750f75-5e10-4e26-9915-470734bef728


 ## Optional suggestions 
N/A

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jovasal committed:
changes requiered under Coder review

Compare your Design to the Original Design for this Project are very different as mentioned by the previous reviewer , your design should refect the original design , Kindly check the vedeo below and match it to the design given for this Project 👍 we are only allowed to change photos and background only

jovasal commented:
all corrections were made successfully.

Munsa1 commented on index.html:
```diff
@@ -0,0 +1,260 @@
+        <link rel="stylesheet" href="style.css">
+        <title>Capstone</title>
+    </head>
+    <body class="bodyart">
+        <header id="header" class="header">
+            <nav class="navbar">
+                <ul class="nav-list">
+                    <li>
+                        <img src="./images/Rectangle.png" class="facebook" alt="facebook"/>
+                    </li>
+                    <li>
+                        <img src="./images/twitter.png" class="facebook" alt="facebook"/>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">English</a>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">My Page</a>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">Logout</a>
```
- [x] As requested by the previous reviewers, kindly ensure that your design matches the given design

### Navigation
Your design
![image](https://github.com/jovasal/Capstone/assets/81932478/bedf018f-5b2e-4b7f-940f-2a8af1b78979)

Given design
![image](https://github.com/jovasal/Capstone/assets/81932478/53004eb8-3474-4a71-ba04-73bc1bfabb5e)

- The top navigation could use some padding so as to add some space around the elements as is in the given design
- The orange box link in the bottom navigation is missing in your design. its the link to the furthest right, kindly add it
![image](https://github.com/jovasal/Capstone/assets/81932478/4b946d65-6e93-42f6-b2fa-67edf316d86d)
- The bottom nav links should not have an underline, and should not use the color black, rather use provided color swatch, and add a hover effect of the orange color to the links
![image](https://github.com/jovasal/Capstone/assets/81932478/c7cb8af0-f5da-41f4-ac13-035555db9921)


### Top Section

Your design
![image](https://github.com/jovasal/Capstone/assets/81932478/d4600fbe-96ff-48f6-9e87-f7bb3171bd9e)

Given design
![image](https://github.com/jovasal/Capstone/assets/81932478/127f5f36-1c29-49e3-944e-282d247a7312)

- Kindly reduce the size of your `h1` tag to `70px` so as to match the given design more.
- Add some margin to the left on all the elements of this section so as to match the given design.
- Also kindly reduce the size of the remaining elements so as to match the given design, for example the paragraph is too bold compared to the given design, and the bold text below the paragraph is too big compared to the given design.


### Speaker Section
Your design
![image](https://github.com/jovasal/Capstone/assets/81932478/4ec2ac2d-097c-4262-a318-d151f58703c3)

Given design
![image](https://github.com/jovasal/Capstone/assets/81932478/9571cd75-3b40-4924-81d9-3cf536db5997)

- Kindly style the speaker section to match the given design, with three speakers on the left and on the right in a top-down fashion.
- The see more button should not be visible in the desktop view.

jovasal replied:
Hi @Munsa1.

A coder review before, wrote that I can change this part and is the only par that wa can change. However, I made changes but, I invite to you to read before to do a code review because you confuse people with your comments.

I made changes as your solicitude now:

![image](https://github.com/jovasal/Capstone/assets/35693330/500dd874-ad5e-4b24-b825-d6d5dac7f980)
![image](https://github.com/jovasal/Capstone/assets/35693330/fc544547-a93e-42b9-a18a-780ff9b68515)

Munsa1 replied:
The changes you have made are great, however to match the given design, the hieght of the image should have been longer e.g

Given design
![image](https://github.com/jovasal/Capstone/assets/81932478/9d26ee48-9bdf-4ded-80e8-fc3078bcf33b)

- Notice how the text to the right does not overflow beyond the bottom of the speaker image?

Munsa1 commented on index.html:
```diff
@@ -0,0 +1,260 @@
+                </div>
+            </div>
+        </section>
+        <section class="main1" id="main1">
+            <div class="firsttemplate1">
+                <div class="info1">
+                    <div class="title11">
+                        <h2 class="title2">The E-Commerce </br>
+                            Pioneers Max 2023
+                        </h2>
```
- [x] (Optional) It would improve the structure of your website if you could separate the index page from the about page by putting the about page in its own `.html` file. Your current structure is hard to read and maintain, thats why the issue of design is a problem. For example the index page is missing a footer and a partner section.

jovasal replied:
ok.

It is a new page:

![image](https://github.com/jovasal/Capstone/assets/35693330/a9e6e262-8088-4458-b617-4f1697e0f5c8)

Munsa1 commented on index.html:
```diff
@@ -0,0 +1,260 @@
+                </div>
+                <div class="RedLine"></div>
+                <div class="SpeakersCont" id="speakersContainer"></div>
+                <div class="SeeMoreButon">
+                    <a href="#" class="SeeMoreBtn BtnR" id="seeMoreBtn">
+                        See More
+                    </a>
+                </div>
+            </div>
+        </section>
```
- [x] After the speaker section, we are supposed to have the `Partner` section and then have the `Footer` for the index page. This is currently missing. Kindly add these sections and please make sure you follow the given design.\

![image](https://github.com/jovasal/Capstone/assets/81932478/567505a6-c118-4b7e-ba85-1da5ccb7316c)

jovasal replied:
Hi Munsa1 Thank you so much for your comments. I really appreciate your knowledges.

However. Footer always ar in the final of the page.

![image](https://github.com/jovasal/Capstone/assets/35693330/cfd12a21-dea4-4c1e-b9ce-0651396d5991)

Munsa1 replied:
- [ ] I was saying that this is a two-page website, it has the home page and the About page. And if you look at the given design they have a structure that involves both pages having a footer section, I was suggesting that you add the footer to the index page.

Munsa1 commented on index.html:
```diff
@@ -0,0 +1,260 @@
+                            Pioneers Max 2023
+                        </h2>
+                    </div>
+                    <div class="squareInfo1">
+                        <h4 class="resumetitle1">
+                            The Global Summit brings together the community
+                            of experts, academics, and activists who comprise 
+                            the Creative Commons affiliate network in a 
+                            different country every two years. This year, 
+                            we hope to expand our invitation list including 
+                            organizations and individuals.
+                        </h4>
+                    </div>
+                    <div class="dateTitle1" id="dateTitle1">
+                        Please contact us per Email for nay further questions about CC Global Summit 2015!
+                    </div>
+                    <div class="siteDir1">ccsummit2015@anaxtasia.com</div>
+                    <div class="ContainerSectionSpeakers">
+                        <div class="SectionSpeakT1">
+                            Creative Commons Global Summit 2015 Logo
+                        </div>
```
- [x] Kindly ensure you also style your About page to match the given design. The major issue is the font size of your headings. Kindly reduce the font size and center your content for the About page as is in the given design

Your design
![image](https://github.com/jovasal/Capstone/assets/81932478/2e183a9f-2687-4a99-9b76-af8b29442715)

Given design
![image](https://github.com/jovasal/Capstone/assets/81932478/003838a7-7dde-463f-a981-9f05d495a0aa)

jovasal replied:
Hi @Munsa1 I changed this color under recomendación mentioned before. Whereas I changed again under your coments.

![image](https://github.com/jovasal/Capstone/assets/35693330/ec6b04cb-481c-4479-96f3-c79a04900920)

Munsa1 replied:
Okay here I meant the font size of your title was too big and therefore was distorting the design, I was suggesting that you significantly reduce the font size

Munsa1 replied:
Also, I meant to suggest that you could align your content to the middle as is in the given design

Munsa1 commented on index.html:
```diff
@@ -0,0 +1,260 @@
+                        </div>
+                        <img id="anaxtasia" src="./images/anaxtasia.jpg" alt="anax"/>
+                            <div class="SectionSpeakT1">
+                                See the past CC Global Summits
+                            </div>                          
+                            <div class="RedLine"></div>
+                            <div class="penul" id="penul">a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</div>
+                            <div class="penul1">
+                                <img id="anaxtasia" src="./images/about_2011 1.png" alt="anax"/>
+                                <img id="anaxtasia" src="./images/about_2013 1.png" alt="anax"/>
+                            </div> 
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
```
- [x] Good job displaying this section horizontally as requested, however kindly add some spacing to the `divs` so as to match the given design

Your design
![image](https://github.com/jovasal/Capstone/assets/81932478/4bc3f35b-f371-4054-aa89-03d9ddcb9c76)

Given design
![image](https://github.com/jovasal/Capstone/assets/81932478/058d1878-e376-4eb5-bf32-3f3ebb3d0c62)

#### Note that the About page has no `Partner` section in the given desktop design, but rather has it in the mobile version.

jovasal replied:
ok. done
![image](https://github.com/jovasal/Capstone/assets/35693330/a88a89a4-1f68-4094-beef-60105dc22888)

Munsa1 reviewed (changes requested):
Hi @jovasal, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

jovasal committed:
Navigation

Added more padding and creat a button CC Campign

jovasal committed:
change all details from code review

done

jovasal committed:
finish

finisg

jovasal committed:
js error fixed

error fixed

brainconnect93 commented on index.html:
```diff
@@ -0,0 +1,241 @@
+                        <a class="header-styles" href="#">Logout</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <section class="section1">
+            <nav class="navbar1">
+                <ul class="header-container">
+                    <img id="anaxtasia" src="./images/anaxtasia.jpg" alt="anax"/>
+                </ul>
+                <ul class="nav-list1">
+                    <li>
+                        <a class="item-styles1" href="./about.html">About</a>
+                    </li>
+                    <li>
+                        <a class="item-styles1" href="#MainProgram">Program</a>
+                    </li>
+                    <li>
+                        <a class="item-styles1" href="#dateTitle1">Join</a>
+                    </li>
+                    <li>
+                        <a class="item-styles1" href="#sponsor">Sponsor</a>
```
- [ ] Great effort working on your project, But I would kindly suggest you stick to the Original design as much as possible and also the suggested changes requested by the previous reviewers, Your project still requires some more attention to meet up with the standard of the original design and having a good design will show the confidence in your ability to replicate any given design by a client. Kindly work on the suggested changes I raised in the video below before resubmitting for review. Thank You.


Video Claim 👇 

https://github.com/jovasal/Capstone/assets/78634841/b31317b3-0ec5-4d6c-b440-b703301897ae

jovasal replied:
Hi, @brainconnect93 thank you so much for your video. unfortunately, I don't understand your English well, could you write me here your comments, please?

jovasal replied:
Hi @brainconnect93. 

Buttons ar working as original design as you recommend on video.

before:

![image](https://github.com/jovasal/Capstone/assets/35693330/58cf98b6-8679-4aae-9a37-b544d9b6ee6d)


now:

![image](https://github.com/jovasal/Capstone/assets/35693330/90529f8a-1301-4fcc-b796-ce84d2e7c98a)

![image](https://github.com/jovasal/Capstone/assets/35693330/847862fe-f644-457f-9af7-9447142bee05)

![image](https://github.com/jovasal/Capstone/assets/35693330/17187950-16da-4cfb-acfa-fac536537d41)

![image](https://github.com/jovasal/Capstone/assets/35693330/8643c719-48c9-4699-a07e-3fb762b17cc3)

![image](https://github.com/jovasal/Capstone/assets/35693330/ecb720ea-056a-413f-b213-c5828f9c5a10)

Now, is equal to original design.

jovasal replied:
Hi @brainconnect93 Thank you so much for your comments about font style and font size, However, if you check all history with other coder review. Al font style, font size and colors were fixed before accorded to other coder review suggested before. I can show you some examples:

![image](https://github.com/jovasal/Capstone/assets/35693330/8d4f2f8b-be1a-49a5-83ff-1d90bc9203b3)
![image](https://github.com/jovasal/Capstone/assets/35693330/37696027-748b-4cb3-bcb4-179fdd15f7e8)

Title and subtitle are ok in fonts and styles. I reduce the sise a legend into square:
![image](https://github.com/jovasal/Capstone/assets/35693330/024b3b33-bb2d-443d-9642-916c82a0e090)

Back ground were fixed:
![image](https://github.com/jovasal/Capstone/assets/35693330/12e0257e-e193-4942-a7d9-49126af3e905)

jovasal replied:
In feature speakers, background color were fixed before as coder review suggested before. Background were fixed with the same color sugested in vehance.net:

![image](https://github.com/jovasal/Capstone/assets/35693330/8a97d34a-6443-4b3f-9448-f0037e090b09)

However, I fix a background again with a soft color similar to design. I justified al text as suggested:

![image](https://github.com/jovasal/Capstone/assets/35693330/7306edec-e633-4e1f-8055-0c9f3f65807b)

jovasal replied:
white space were deleted as suggested in video:

![image](https://github.com/jovasal/Capstone/assets/35693330/545cfc1b-2689-4fa8-9121-27c48ae72bd1)


Now, Design si:

![image](https://github.com/jovasal/Capstone/assets/35693330/04c06827-1bb3-46da-accd-99d3042f7855)

jovasal replied:
Partner were fixes as suggested before:

![image](https://github.com/jovasal/Capstone/assets/35693330/51420b6c-a426-4ca6-a3e9-302970b59579)

jovasal replied:
al pages were made in differents sections:

![image](https://github.com/jovasal/Capstone/assets/35693330/bcbeef5a-e235-419e-907b-3fe688fe3640)

![image](https://github.com/jovasal/Capstone/assets/35693330/b2dc260a-53a7-4472-8046-7f287afe8c4a)

![image](https://github.com/jovasal/Capstone/assets/35693330/1a0deeb0-8d4d-4e00-ae76-ae557b283d89)
<section class="main1" id="main1">
            <div class="firsttemplate1">
                <div class="info1">
                    <div class="title11">
                        <h2 class="title21">The E-Commerce </br>
                            Pioneers Max 2023
                        </h2>
                    </div>
                    <div class="squareInfo1">
                        <h4 class="resumetitle1">
                            The Global Summit brings together the community
                            of experts, academics, and activists who comprise 
                            the Creative Commons affiliate network in a 
                            different country every two years. This year, 
                            we hope to expand our invitation list including 
                            organizations and individuals.
                        </h4>
                    </div>
                    <div class="dateTitle1" id="dateTitle1">
                        Please contact us per Email for nay further questions about CC Global Summit 2015!
                    </div>
                    <div class="siteDir1">
                        ccsummit2015@anaxtasia.com
                    </div>
                </div>
            </div>
        </section>
        <section class="main11" id="main11">
            <div class="firsttemplate11">
                <div class="info1">
                    <div class="ContainerSectionSpeakers">
                        <div class="SectionSpeakT1">
                            Creative Commons Global Summit 2015 Logo
                        </div>
                        <div class="RedLine"></div>
                        <div class="dateTitle11">
                            The logo of CC Global Summit 2015 was 
                            decided through the logo competition 
                            from 8, June to 7, July.
                        </div>
                        <img id="anaxtasia" src="./images/anaxtasia.jpg" alt="anax"/>
                    </div>
                </div>
            </div>
        </section>
        <section class="main11" id="main11">
            <div class="firsttemplate11">
                <div class="info1">
                    <div class="SectionSpeakT1">
                        See the past CC Global Summits
                    </div>
                    <div class="RedLine"></div>
                    <div class="penul" id="penul">a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</div>
                    <div class="penul1">
                        <div class="pict1">
                            <img id="anaxtasia" src="./images/about_2011 1.png" alt="anax"/>
                        </div>
                        <div class="pict2">
                            <img id="anaxtasia" src="./images/about_2013 1.png" alt="anax"/>
                        </div>
                    </div>
                </div>
            </div>
        </section>

jovasal replied:
Thank you so much!!! you have all reazon about (About Us) section.

![image](https://github.com/jovasal/Capstone/assets/35693330/c1bffec9-e75e-4db7-98c3-9ab1fecb0e57)

I changed with the same design and is working fine. All buttons are working and is clear to use.

![image](https://github.com/jovasal/Capstone/assets/35693330/3362a151-d094-4182-aadb-bcca85811b5d)

jovasal replied:
Thank you so much @Munsa1 @brainconnect93 @Cabraham1 @danmainah 

This is my last opportunity to pass. I'm sure I made all recommended changes and in an excellent copy of the original design.

brainconnect93 reviewed (changes requested):
## STATUS: Changes Required :recycle:

### Hi @jovasal, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media.tenor.com/isiLRnxz3zwAAAAM/cat-driving-serious.gif" width="500px" height="300px">

## To Highlight 👍

- Descriptive Pull Request title and summary. ✔️
- Project contains a Home & About page ✔️
- hamburger menu on the left side ✔️
- Follow [Cindy Shin](https://www.behance.net/adagio07) design template ✔️
- Good use of Github flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jovasal committed:
All changes were made under recomendatios for all code reviewers.

Great copy now

AbayomiOlaoye commented on index.html:
```diff
@@ -0,0 +1,277 @@
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="main11" id="main11">
+            <div class="firsttemplate11">
+                <div class="info1">
+                    <div class="ContainerSectionSpeakers">
+                        <div class="SectionSpeakT1">
+                            Creative Commons Global Summit 2015 Logo
+                        </div>
+                        <div class="RedLine"></div>
+                        <div class="dateTitle11">
+                            The logo of CC Global Summit 2015 was 
+                            decided through the logo competition 
+                            from 8, June to 7, July.
+                        </div>
+                        <img id="anaxtasia" src="./images/anaxtasia.jpg" alt="anax"/>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="main11" id="main11">
+            <div class="firsttemplate11">
+                <div class="info1">
+                    <div class="SectionSpeakT1">
+                        See the past CC Global Summits
+                    </div>
+                    <div class="RedLine"></div>
+                    <div class="penul" id="penul">a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</div>
+                    <div class="penul1">
+                        <div class="pict1">
+                            <img id="anaxtasia" src="./images/about_2011 1.png" alt="anax"/>
+                        </div>
+                        <div class="pict2">
+                            <img id="anaxtasia" src="./images/about_2013 1.png" alt="anax"/>
+                        </div>
+                    </div>
+                </div>
+            </div>
```
- You are close, my friend! :muscle: 
- [ ] I am not sure that these highlighted sections should be here in the `Home` page. Kindly check Cindy's design [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and move it to the `About` page please.
- [ ] Also, I think you should watch the attached [video](https://watch.screencastify.com/v/UgXfOlXJAOKJFbwzGKif) if you haven't, please. It will save you of some troubles :pray:

jovasal replied:
Hi @AbayomiOlaoye Thank you so much for your time. I'm doing the suggested changes for you and I found some differences with other code reviews that checked before. However. I made the changes suggested for you.

@brainconnect93 suggested reducing a size
![image](https://github.com/jovasal/Capstone/assets/35693330/4243271d-d5fb-457d-b58e-2164879d4ead)
Now I increase again and I reduce the images as suggested for you:
![image](https://github.com/jovasal/Capstone/assets/35693330/7a466af9-76e8-4d14-8f91-544e3ae2c193)

jovasal replied:
Also, something was wrong when I saved and moved a button. Now is at the same level as you can see:

              
![image](https://github.com/jovasal/Capstone/assets/35693330/17dcd6dc-b369-4409-8879-4e503e178715)

jovasal replied:
Margin changed:

before:
![image](https://github.com/jovasal/Capstone/assets/35693330/c0de1eb2-0863-481a-a54b-152343f068f9)
now:
![image](https://github.com/jovasal/Capstone/assets/35693330/433aab2b-8a26-4d50-85a9-665483e238cb)

jovasal replied:
before:
![image](https://github.com/jovasal/Capstone/assets/35693330/9cc82847-1de4-412c-b346-83957b843d28)

now:
![image](https://github.com/jovasal/Capstone/assets/35693330/6200f43c-4233-4a6b-82ab-e34c8063fed1)

jovasal replied:
![image](https://github.com/jovasal/Capstone/assets/35693330/0cf8c794-99c0-496e-bcbf-9f42212e0dcb)

jovasal replied:
About, is on a different page, because About is like that:

![image](https://github.com/jovasal/Capstone/assets/35693330/acd255d7-fdd5-4e0a-a2a2-771c670166ea)


for that reason, about is only on one page:
![image](https://github.com/jovasal/Capstone/assets/35693330/180c12aa-c567-42ee-8f43-0d3d1b7933b3)

jovasal replied:
On another page, I put design and @AbayomiOlaoye suggested, but no in About. is in a different page:
design is:
![image](https://github.com/jovasal/Capstone/assets/35693330/e58eaf68-5ff8-48ad-ba8b-21835fb5ab9c)
Now, my page is:
![image](https://github.com/jovasal/Capstone/assets/35693330/25c3764d-4d95-42f9-80f3-e22b213adf9c)

jovasal replied:
this is my last extra review and the project are working fine. please, help me because is I do not pass, I can follow my program.

I am sure that my page is working fine. Also, buttons are matched with each part of the project as suggested @AbayomiOlaoye

topeogunleye replied:
> * You are close, my friend! 💪
> * [ ]  I am not sure that these highlighted sections should be here in the `Home` page. Kindly check Cindy's design [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and move it to the `About` page please.
> * [ ]  Also, I think you should watch the attached [video](https://watch.screencastify.com/v/UgXfOlXJAOKJFbwzGKif) if you haven't, please. It will save you of some troubles 🙏

@jovasal, Kindly note that the code reviewer says here that the h

> * ot sure that these highlighted sections should be here in the `Home` page. Kindly check Cindy's design [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and move it to the `About` page please.

@jovasal Kindly read the statement of this reviewer saying that these highlighted sections should not be on the `Home` page, and Kindly check Cindy's design [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and move it to the `About` page, please.

You removed the highlighted sections from the home page, but you didn't move them to the about page

AbayomiOlaoye reviewed (changes requested):
## STATUS: CHANGE REQUIRED ♻️ 

Hi @jovasal :wave: :wave: , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![invalid](https://user-images.githubusercontent.com/99715640/211858554-39090b6b-2f0f-45dd-b5e0-8fb805e56431.gif)

## Highlights
- Follow Gitflow :heavy_check_mark: 
- Descriptive ReadMe :heavy_check_mark: 
- Good attitude to feedback :heavy_check_mark: 
- Your determination to make things work is great :heavy_check_mark: 

## Suggested changes
- [ ] _Check the comments under the review._
- [ ] I think the texts might be confusing and so I have recorded a video guide for you [here](https://watch.screencastify.com/v/UgXfOlXJAOKJFbwzGKif). Please watch it now.
- [ ] After seeing the video, kindly go through the past reviews now and see what the concerns of my colleagues were in trying to help you create a smashing conference page. Then make appropriate changes.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

jovasal commented:
Y dont hace More changes no. Thank you

El sáb., 8 jul. 2023, 1:17 p. m., Olaoye Oluwatoyin Abayomi <
***@***.***> escribió:

> ***@***.**** requested changes on this pull request.
> STATUS: CHANGE REQUIRED ♻️
>
> Hi @jovasal <https://github.com/jovasal> 👋 👋 ,
>
> Great work on making the changes 👏
> You've done well implementing some of the requested changes, but there are
> still some which aren't addressed yet.
>
> [image: invalid]
> <https://user-images.githubusercontent.com/99715640/211858554-39090b6b-2f0f-45dd-b5e0-8fb805e56431.gif>
> Highlights
>
>    - Follow Gitflow ✔️
>    - Descriptive ReadMe ✔️
>    - Good attitude to feedback ✔️
>    - Your determination to make things work is great ✔️
>
> Suggested changes
>
>    - *Check the comments under the review.*
>    - I think the texts might be confusing and so I have recorded a video
>    guide for you here
>    <https://watch.screencastify.com/v/UgXfOlXJAOKJFbwzGKif>. Please watch
>    it now.
>    - After seeing the video, kindly go through the past reviews now and
>    see what the concerns of my colleagues were in trying to help you create a
>    smashing conference page. Then make appropriate changes.
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> In index.html
> <https://github.com/jovasal/Capstone/pull/1#discussion_r1257323908>:
>
> > +        <section class="main1" id="main1">
> +            <div class="firsttemplate1">
> +                <div class="info1">
> +                    <div class="title11">
> +                        <h2 class="title21">The E-Commerce </br>
> +                            Pioneers Max 2023
> +                        </h2>
> +                    </div>
> +                    <div class="squareInfo1">
> +                        <h4 class="resumetitle1">
> +                            The Global Summit brings together the community
> +                            of experts, academics, and activists who comprise
> +                            the Creative Commons affiliate network in a
> +                            different country every two years. This year,
> +                            we hope to expand our invitation list including
> +                            organizations and individuals.
> +                        </h4>
> +                    </div>
> +                    <div class="dateTitle1" id="dateTitle1">
> +                        Please contact us per Email for nay further questions about CC Global Summit 2015!
> +                    </div>
> +                    <div class="siteDir1">
> +                        ***@***.***
> +                    </div>
> +                </div>
> +            </div>
> +        </section>
> +        <section class="main11" id="main11">
> +            <div class="firsttemplate11">
> +                <div class="info1">
> +                    <div class="ContainerSectionSpeakers">
> +                        <div class="SectionSpeakT1">
> +                            Creative Commons Global Summit 2015 Logo
> +                        </div>
> +                        <div class="RedLine"></div>
> +                        <div class="dateTitle11">
> +                            The logo of CC Global Summit 2015 was
> +                            decided through the logo competition
> +                            from 8, June to 7, July.
> +                        </div>
> +                        <img id="anaxtasia" src="./images/anaxtasia.jpg" alt="anax"/>
> +                    </div>
> +                </div>
> +            </div>
> +        </section>
> +        <section class="main11" id="main11">
> +            <div class="firsttemplate11">
> +                <div class="info1">
> +                    <div class="SectionSpeakT1">
> +                        See the past CC Global Summits
> +                    </div>
> +                    <div class="RedLine"></div>
> +                    <div class="penul" id="penul">a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</div>
> +                    <div class="penul1">
> +                        <div class="pict1">
> +                            <img id="anaxtasia" src="./images/about_2011 1.png" alt="anax"/>
> +                        </div>
> +                        <div class="pict2">
> +                            <img id="anaxtasia" src="./images/about_2013 1.png" alt="anax"/>
> +                        </div>
> +                    </div>
> +                </div>
> +            </div>
>
>
>    - You are close, my friend! 💪
>    - I am not sure that these highlighted sections should be here in the
>    Home page. Kindly check Cindy's design here
>    <https://www.behance.net/gallery/29845175/CC-Global-Summit-2015> and
>    move it to the About page please.
>    - Also, I think you should watch the attached video
>    <https://watch.screencastify.com/v/UgXfOlXJAOKJFbwzGKif> if you
>    haven't, please. It will save you of some troubles 🙏
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jovasal/Capstone/pull/1#pullrequestreview-1520749617>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIQKGEXXPI2UPVVNGEXI7I3XPGP5LANCNFSM6AAAAAAZ4WYZHQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

jovasal committed:
several changed made by coder reviwer sugested

ok

jovasal commented:
I don't have more chance. This is my last extra review please help me. If I do another extra review, I lost my chance to continue in Microverse.

I'm sure that my code is ok

topeogunleye commented on about.html:
```diff
@@ -0,0 +1,164 @@
+                </ul>
+            </nav>
+        </header>
+        <section class="section1">
```
- [ ] As suggested by the previous reviewer, the parts of your code that he highlighted should be moved to your about page. Please kindly watch this [video](https://watch.screencastify.com/v/UgXfOlXJAOKJFbwzGKif) by the previous reviewer to understand more.

jovasal replied:
hi
I don't understand your answer.
As suggested by the previous reviewer, the parts of your code that h
![2023_07_08_21_10_52_Window](https://github.com/jovasal/Capstone/assets/35693330/d8d03b62-640a-41c8-8d1c-5bd44917d6ba)
e highly
![2023_07_08_21_10_06_HTML_CSS_JavaScript_capstone_project_Conference_page_by_jovasal_Pull_Reque](https://github.com/jovasal/Capstone/assets/35693330/04ab26b6-3f47-4445-bb5f-69e833382297)
ghted should be moved to your About page. Please kindly watch this [video](https://watch.screencastify.com/v/UgXfOlXJAOKJFbwzGKif) by the previous reviewer to understand more.
I create an About page. The About page is not the design, I create another page for this purpose and I explain it in my pull request.
Please, recheck it

If you check the design. About is my creation, and is not in design. as @AbayomiOlaoye said. he believes, but I'm sure that is not in that part and I explain that.

I think that you made a mistake and need to chek

topeogunleye reviewed (changes requested):
Hi @jovasal, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![Try-Again](https://user-images.githubusercontent.com/52934128/180646051-aed7efa6-6d15-4853-9962-73186e366f2c.gif)

## Highlights
- Tried to make changes requested by the previous reviewer. ✔️ 

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jovasal commented:
should be moved is not true. @AbayomiOlaoye he believes that needs to move to about. Is not true. I create another for this part.

topeogunleye commented:
> should be moved is not true. @AbayomiOlaoye he believes that needs to move to about. Is not true. I create another for this part.
The part you moved to the creative.html page should be in your about.html. 

<img width="952" alt="image" src="https://github.com/jovasal/Capstone/assets/52934128/7d78f6ff-554a-48f7-8fef-dcc056673313">


I encourage you to ask for a second opinion  using [this form](https://airtable.com/shrQAqnBwek5a0O0s) if you feel the review is unfair

jovasal commented:
Hi.

Thank you so much for your feedback. However, Is necesary that you study
all design.

If you check that, you can see that code Is too specific, And exercise is
clear. You need to do a copy of the code.

If you see, about not exist in code. Is a extra part of the code. However,
if you see al design Is clear that design contains Two specific partes, and
you can't change the design, because is a copy.

For that reason. You can't change a part of the specific code. You need to
do a code as similar as posible.

Now, if you show me that design white about part, I finish my conversación.
If not. I think that you need to reoeat the fiesta module, because you dont
understand the project.

Finally. If you check a previos video Sent by last coser Review. He said
Cleary. I BELIEVE not YOU MUST.

I Sent a requirentes to Microverse, because you ate blocking my Advance And
you know that Is true.

Thank you And Beat Regards.

El sáb., 8 jul. 2023, 9:53 p. m., Temitope Ogunleye <
***@***.***> escribió:

> should be moved is not true. @AbayomiOlaoye
> <https://github.com/AbayomiOlaoye> he believes that needs to move to
> about. Is not true. I create another for this part.
> The part you moved to the creative.html page should be in your about.html.
>
> img width="952" alt="image" src="
> https://github.com/jovasal/Capstone/assets/52934128/7d78f6ff-554a-48f7-8fef-dcc056673313">
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jovasal/Capstone/pull/1#issuecomment-1627582108>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIQKGERBAMK6O7K6IZU5ZX3XPIMI5ANCNFSM6AAAAAAZ4WYZHQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

topeogunleye commented:
> Hi. Thank you so much for your feedback. However, Is necesary that you study all design. If you check that, you can see that code Is too specific, And exercise is clear. You need to do a copy of the code. If you see, about not exist in code. Is a extra part of the code. However, if you see al design Is clear that design contains Two specific partes, and you can't change the design, because is a copy. For that reason. You can't change a part of the specific code. You need to do a code as similar as posible. Now, if you show me that design white about part, I finish my conversación. If not. I think that you need to reoeat the fiesta module, because you dont understand the project. Finally. If you check a previos video Sent by last coser Review. He said Cleary. I BELIEVE not YOU MUST. I Sent a requirentes to Microverse, because you ate blocking my Advance And you know that Is true. Thank you And Beat Regards. El sáb., 8 jul. 2023, 9:53 p. m., Temitope Ogunleye < ***@***.***> escribió:
> […](#)
> should be moved is not true. @AbayomiOlaoye <https://github.com/AbayomiOlaoye> he believes that needs to move to about. Is not true. I create another for this part. The part you moved to the creative.html page should be in your about.html. img width="952" alt="image" src=" https://github.com/jovasal/Capstone/assets/52934128/7d78f6ff-554a-48f7-8fef-dcc056673313"> — Reply to this email directly, view it on GitHub <[#1 (comment)](https://github.com/jovasal/Capstone/pull/1#issuecomment-1627582108)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AIQKGERBAMK6O7K6IZU5ZX3XPIMI5ANCNFSM6AAAAAAZ4WYZHQ> . You are receiving this because you were mentioned.Message ID: ***@***.***>

I'm sorry you feel blocked. The reviewer said I believe because Reviewers are to give suggestions and not order

jovasal commented:
Ok. This Is my first Time in my life that I believe Is the same you should.
Al days I learn something new!

I think, Is posible, could..... Several words! But BELIEVE?

I think that you need to consider your desision, because your answers are
showing your mistake in a crucial parte of project.

Have a good Time!


El sáb., 8 jul. 2023, 10:29 p. m., Temitope Ogunleye <
***@***.***> escribió:

> Hi. Thank you so much for your feedback. However, Is necesary that you
> study all design. If you check that, you can see that code Is too specific,
> And exercise is clear. You need to do a copy of the code. If you see, about
> not exist in code. Is a extra part of the code. However, if you see al
> design Is clear that design contains Two specific partes, and you can't
> change the design, because is a copy. For that reason. You can't change a
> part of the specific code. You need to do a code as similar as posible.
> Now, if you show me that design white about part, I finish my conversación.
> If not. I think that you need to reoeat the fiesta module, because you dont
> understand the project. Finally. If you check a previos video Sent by last
> coser Review. He said Cleary. I BELIEVE not YOU MUST. I Sent a requirentes
> to Microverse, because you ate blocking my Advance And you know that Is
> true. Thank you And Beat Regards. El sáb., 8 jul. 2023, 9:53 p. m.,
> Temitope Ogunleye < *@*.
>
> *> escribió: … <#m_-6185484750960299199_> should be moved is not true.
> @AbayomiOlaoye <https://github.com/AbayomiOlaoye>
> https://github.com/AbayomiOlaoye <https://github.com/AbayomiOlaoye> he
> believes that needs to move to about. Is not true. I create another for
> this part. The part you moved to the creative.html page should be in your
> about.html. img width="952" alt="image" src="
> https://github.com/jovasal/Capstone/assets/52934128/7d78f6ff-554a-48f7-8fef-dcc056673313">
> <https://github.com/jovasal/Capstone/assets/52934128/7d78f6ff-554a-48f7-8fef-dcc056673313%22%3E>
> — Reply to this email directly, view it on GitHub <#1 (comment)
> <https://github.com/jovasal/Capstone/pull/1#issuecomment-1627582108>>, or
> unsubscribe
> https://github.com/notifications/unsubscribe-auth/AIQKGERBAMK6O7K6IZU5ZX3XPIMI5ANCNFSM6AAAAAAZ4WYZHQ
> <https://github.com/notifications/unsubscribe-auth/AIQKGERBAMK6O7K6IZU5ZX3XPIMI5ANCNFSM6AAAAAAZ4WYZHQ>
> . You are receiving this because you were mentioned.Message ID: @.*>
>
> I'm sorry you feel blocked. The reviewer said I believe because Reviewers
> are to give suggestions and not order
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jovasal/Capstone/pull/1#issuecomment-1627589720>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIQKGESVSFN36LVGEJRLVZLXPIQQBANCNFSM6AAAAAAZ4WYZHQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

topeogunleye commented:
> Ok. This Is my first Time in my life that I believe Is the same you should. Al days I learn something new! I think, Is posible, could..... Several words! But BELIEVE? I think that you need to consider your desision, because your answers are showing your mistake in a crucial parte of project. Have a good Time! El sáb., 8 jul. 2023, 10:29 p. m., Temitope Ogunleye < ***@***.***> escribió:
> […](#)
> Hi. Thank you so much for your feedback. However, Is necesary that you study all design. If you check that, you can see that code Is too specific, And exercise is clear. You need to do a copy of the code. If you see, about not exist in code. Is a extra part of the code. However, if you see al design Is clear that design contains Two specific partes, and you can't change the design, because is a copy. For that reason. You can't change a part of the specific code. You need to do a code as similar as posible. Now, if you show me that design white about part, I finish my conversación. If not. I think that you need to reoeat the fiesta module, because you dont understand the project. Finally. If you check a previos video Sent by last coser Review. He said Cleary. I BELIEVE not YOU MUST. I Sent a requirentes to Microverse, because you ate blocking my Advance And you know that Is true. Thank you And Beat Regards. El sáb., 8 jul. 2023, 9:53 p. m., Temitope Ogunleye < *@*. *> escribió: … <#m_-6185484750960299199_> should be moved is not true. @AbayomiOlaoye <https://github.com/AbayomiOlaoye> https://github.com/AbayomiOlaoye <https://github.com/AbayomiOlaoye> he believes that needs to move to about. Is not true. I create another for this part. The part you moved to the creative.html page should be in your about.html. img width="952" alt="image" src=" https://github.com/jovasal/Capstone/assets/52934128/7d78f6ff-554a-48f7-8fef-dcc056673313"> <https://github.com/jovasal/Capstone/assets/52934128/7d78f6ff-554a-48f7-8fef-dcc056673313%22%3E> — Reply to this email directly, view it on GitHub <#1 (comment) <[#1 (comment)](https://github.com/jovasal/Capstone/pull/1#issuecomment-1627582108)>>, or unsubscribe https://github.com/notifications/unsubscribe-auth/AIQKGERBAMK6O7K6IZU5ZX3XPIMI5ANCNFSM6AAAAAAZ4WYZHQ <https://github.com/notifications/unsubscribe-auth/AIQKGERBAMK6O7K6IZU5ZX3XPIMI5ANCNFSM6AAAAAAZ4WYZHQ> . You are receiving this because you were mentioned.Message ID: @.*> I'm sorry you feel blocked. The reviewer said I believe because Reviewers are to give suggestions and not order — Reply to this email directly, view it on GitHub <[#1 (comment)](https://github.com/jovasal/Capstone/pull/1#issuecomment-1627589720)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AIQKGESVSFN36LVGEJRLVZLXPIQQBANCNFSM6AAAAAAZ4WYZHQ> . You are receiving this because you were mentioned.Message ID: ***@***.***>

If a code reviewer has not marked the change as optional, it should be done. If you have any doubts, feel free to request a second opinion

Also: This is the previous statement of the code reviewer I was referring to that you didn't make the change requested. He clearly ask's you to `Kindly check Cindy's design [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and move it to the About page please` :
<img width="584" alt="image" src="https://github.com/jovasal/Capstone/assets/52934128/a482fa1b-92ac-401d-9a66-44dc8eb06e81">

jovasal commented:
Hi, @topeogunleye ok. I checked Cindy's design and can you tell me where is about page in creative commons?

![image](https://github.com/jovasal/Capstone/assets/35693330/e6dc9245-db4a-4c6e-84c4-5bdf9be90c44)


Great!!! in not here!!! you know, in which part about should be used?

Great!! About page does not exist in Cyndy's design. For that reason, you can't change a design shown before. The only way is to create a new page for that because if you paste about section in creative commons, you change the real design.

I invite you to read the exercise and also, is necessary that you read my answer before.

![image](https://github.com/jovasal/Capstone/assets/35693330/f325021c-8c68-46a1-b6b7-f58552f8e5d6)

Now. if you are confused with (I believe) because for you this word is mandatory, I can understand that you can't process this information. I think that you need to study again and I talked with Microverse. You are not ready for a code review and you don't understand that you need to check all information. Maybe, I'm not continuing in Microverse because my experience with you is too bad. but believe me, you need to be prepared for a code review or Microverse needs to change you. I'm working on that.

Is not necessary that you answer this message. Thank you and Best regards.

topeogunleye commented:
> e confused with (I believe) because for you this word is mandatory, I can unde



> Hi, @topeogunleye ok. I checked Cindy's design and can you tell me where is about page in creative commons?
> 
> ![image](https://user-images.githubusercontent.com/35693330/252186673-e6dc9245-db4a-4c6e-84c4-5bdf9be90c44.png)
> 
> Great!!! in not here!!! you know, in which part about should be used?
> 
> Great!! About page does not exist in Cyndy's design. For that reason, you can't change a design shown before. The only way is to create a new page for that because if you paste about section in creative commons, you change the real design.
> 
> I invite you to read the exercise and also, is necessary that you read my answer before.
> 
> ![image](https://user-images.githubusercontent.com/35693330/252187341-f325021c-8c68-46a1-b6b7-f58552f8e5d6.png)
> 
> Now. if you are confused with (I believe) because for you this word is mandatory, I can understand that you can't process this information. I think that you need to study again and I talked with Microverse. You are not ready for a code review and you don't understand that you need to check all information. Maybe, I'm not continuing in Microverse because my experience with you is too bad. but believe me, you need to be prepared for a code review or Microverse needs to change you. I'm working on that.
> 
> Is not necessary that you answer this message. Thank you and Best regards.


I believe that you misunderstood the meaning of “kindly” in the context of the message. [“Kindly” is used as an adverb to politely request someone for something](https://www.dictionary.com/browse/kindly)[1](https://www.dictionary.com/browse/kindly). [In this case](https://github.com/jovasal/Capstone/pull/1#discussion_r1257323908), [the previous code reviewer was politely requesting you to check Cindy’s design and move it to the About page](https://educalingo.com/en/dic-en/kindly)[2](https://github.com/jovasal/Capstone/pull/1#discussion_r1257323908). I hope this clears up any confusion.

https://github.com/jovasal/Capstone/pull/1#discussion_r1257323908


The project requirements clearly state that you should build only two pages: the home page and the About page. I hope this clears up any confusion.

<img width="630" alt="image" src="https://github.com/jovasal/Capstone/assets/52934128/f668bb13-4dbd-49ca-8100-75b4037972a0">

topeogunleye commented:
@jovasal, Here is the `About` page from Cindy's design:


![image](https://github.com/jovasal/Capstone/assets/52934128/c61af873-0517-4145-921b-aff43afc49b8)

jovasal commented:
Hi, @topeogunleye. About page was on the home page before. That is great, I can choose where I can put it because any page does not exist (about) if I put (about in any of that pages, I change all designs) so, the problem is that I had about page in about page along ant the rest of the code in home page. Now, another code review suggested putting a part of the code in about page changing all original designs. It is not possible to do, and this is the point that you don't understand. I invite you to check all details.

![image](https://github.com/jovasal/Capstone/assets/35693330/ddde4daa-41c2-4cc0-985d-2060643304d0)

All changes were taken under code review before, so, If I take a decision to paste the rest of the code into the About page, is a clear mistake, because the About page does not exist in the code. You can see that the About page does not exist in:

![image](https://github.com/jovasal/Capstone/assets/35693330/2347903e-b4f7-4b14-9cc9-7f5f2a08fb4d)


So, the problem comes in suggested changes before and you don't understand this problem. also, your confusion with I believe as a mandatory word is a big mistake.

nicupop729 commented:
This PR is under a Second Opinion review.

Kindly, wait for the decision.

nicupop729 reviewed (commented):
Hello 👋,

_This is an additional review as a result of the second opinion request._

Please note that, as mentioned by the previous reviewers, you still need to implement the About page for both mobile and desktop views. When the user clicks on the About button in the navigation, the page should display the provided About page design.
![image](https://github.com/jovasal/Capstone/assets/83596135/cd57849b-9728-4f07-a863-ecc62e026246)
![image](https://github.com/jovasal/Capstone/assets/83596135/11698286-8dea-4e7c-a297-80818439bac1)

Considering that you did not receive approval last week, you will need to continue working on the same project this week as well.

Please note that it is not allowed to use the same PR to request another review this week. In order to ask for a new review this week, you need to create a completely new PR. This allows the first reviewer to provide a fresh review without considering the feedback from previous reviewers.

Since the mobile view requires significant adjustments to match the given design and the About page still needs to be built for both mobile and desktop view, I suggest building the project using a completely new repository. However, if you wish, you can still use the same repository, but it is not allowed to use the same Pull Request.

I kindly request you to make an effort to build the project as shown in the given design. When you are ready, please request another review from your dashboard!

Keep rocking 🚀!

Reviewers, please take note that if a project is not approved in a particular week, it is not permissible to review the same PR in the following week. Instead, students must use a completely new PR or even a different repository. For more information on this matter, please refer to the FAQ section [here](https://gitlab.com/microverse/guides/tse/how_to_be_a_tse/-/blob/master/faq/pull_request_issues.md#if-the-student-is-repeating-the-blockmodule-previously-unapproved-pr).
